examples/quickstart.php

```diff
@@ -73,25 +76,20 @@
 $eventStore->create($singleStream);
 
 /**
- * Now we can easily add events to the stream ...
- */
-$eventStore->appendTo($streamName, new ArrayIterator([$quickStartSucceeded /*, ...*/]));
-
-/**
  * Next step would be to commit the transaction.
  * But let's attach a plugin first that prints some information about currently added events.
  * Plugins are simple event listeners. See the docs of prooph/common for more details about event listeners.
  */
 $eventStore->getActionEventEmitter()->attachListener(
-    'commit.post', //Most of the event store methods provide pre and post hooks
-    function (\Prooph\Common\Event\ActionEvent $actionEvent): void {
+    ActionEventEmitterAware::EVENT_APPEND_TO, // InMemoryAdapter provides event hooks
+    function (ActionEvent $actionEvent): void {
         /**
          * In the *commit.post* action event a plugin has access to
          * all recorded events which were added in the current committed transaction.
          * It is the ideal place to attach a domain event dispatcher.
          * We only use a closure here to print the recorded events in the terminal
          */
-        $recordedEvents = $actionEvent->getParam('recordedEvents');
+        $recordedEvents = $actionEvent->getParam('streamEvents');
```

Comment:
We should collect and list all BC breaks, also the small ones like changed action event param names.